common/lib/capa/capa/templates/codeinput.html

```diff
@@ -36,6 +39,6 @@
   <span id="answer_${id}"></span>
 
   <div class="external-grader-message" aria-live="polite">
-    ${msg|n}
+    ${HTML(msg)}
```

Comment:
@cptvitamin @sstack22 has an accessibility review been done of the External Grader (http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/external_graders.html?highlight=coderesponse)? I imagine there are all sorts of accessibility issues with it, given how it presents feedback from the grader.

Here is a working example:
https://courses.edx.org/courses/edX/DemoX.1/2014/courseware/70a1e3505d83411bb72393048ac4afd8/0775ef77335d4b2994d2d9bd2e0ce8c1/4

FYI @staubina, who is also struggling with this problem type (focus test).